mne/channels/data/montages/standard_1020-62.elc

```diff
@@ -0,0 +1,136 @@
+# ASA electrode file
+ReferenceLabel	avg
+UnitPosition	mm
+NumberPositions=	65
+Positions
+-86.0761 -19.9897 -47.9860
+85.7939 -20.0093 -48.0310
+0.0083 86.8110 -39.9830
+-29.4367 83.9171 -6.99
+0.1123 88.247 -1.713
+29.8723 84.8959 -7.08
+-33.7007 76.8371 21.227
+35.7123 77.7259 21.956
+-70.2629 42.4743 -11.42
+-64.4658 48.0353 16.921
+-50.2438 53.1112 42.192
+-27.4958 56.9311 60.342
+0.3122 58.512 66.462
+29.5142 57.6019 59.54
+51.8362 54.3048 40.814
+67.9142 49.8297 16.367
+73.0431 44.4217 -12
+-80.775 14.1203 -11.135
+-77.2149 18.6433 24.46
+-60.1819 22.7162 55.544
+-34.0619 26.0111 79.987
+0.3761 27.39 88.668
+34.7841 26.4379 78.808
+62.2931 23.7228 55.63
+79.5341 19.9357 24.438
+81.8151 15.4167 -11.33
+-84.1611 -16.0187 -9.346
+-80.2801 -13.7597 29.16
+-65.3581 -11.6317 64.358
+-36.158 -9.9839 89.752
+0.4009 -9.167 100.244
+37.672 -9.6241 88.412
+67.1179 -10.9003 63.58
+83.4559 -12.7763 29.208
+85.0799 -15.0203 -9.49
+-84.8302 -46.0217 -7.056
+-79.5922 -46.5507 30.949
+-63.5562 -47.0088 65.624
+-35.5131 -47.2919 91.315
+0.3858 -47.318 99.432
+38.3838 -47.0731 90.695
+66.6118 -46.6372 65.58
+83.3218 -46.1013 31.206
+85.5488 -45.5453 -7.13
+-72.4343 -73.4527 -2.487
+-67.2723 -76.2907 28.382
+-53.0073 -78.7878 55.94
+-28.6203 -80.5249 75.436
+0.3247 -81.115 82.615
+31.9197 -80.4871 76.716
+55.6667 -78.5602 56.561
+67.8877 -75.9043 28.091
+73.0557 -73.0683 -2.54
+-54.8404 -97.5279 2.792
+-48.4244 -99.3408 21.599
+-36.5114 -100.8529 37.167
+0.2156 -102.178 50.608
+36.7816 -100.8491 36.397
+49.8196 -99.4461 21.727
+55.6666 -97.6251 2.73
+-54.9104 -98.0448 -35.465
+-29.4134 -112.449 8.839
+0.1076 -114.892 14.657
+29.8426 -112.156 8.8
+54.9876 -98.0911 -35.541
+Labels
+LPA
+RPA
+Nz
+Fp1
+Fpz
+Fp2
+AF3
+AF4
+F7
+F5
+F3
+F1
+Fz
+F2
+F4
+F6
+F8
+FT7
+FC5
+FC3
+FC1
+FCz
+FC2
+FC4
+FC6
+FT8
+T7
+C5
+C3
+C1
+Cz
+C2
+C4
+C6
+T8
+TP7
+CP5
+CP3
+CP1
+CPz
+CP2
+CP4
+CP6
+TP8
+P7
+P5
+P3
+P1
+Pz
+P2
+P4
+P6
+P8
+PO7
+PO5
+PO3
+POz
+PO4
+PO6
+PO8
+Cb1
```

Comment:
As far as I know Cb1 and Cb2 are not part of the 10-10, 10-20, or 10-05 systems - what are these?

Reply:
See the plot in my comment below.

Reply:
Ah okay, I am still confused though, the positions of Cb2 here for example should rather be something like POO10h or OI2 according to the 10-05 system. Perhaps these are some "special" positions (exceptions to the 10-05 rule)?


https://github.com/sappelhoff/eeg_positions#10-05-system

![image](https://user-images.githubusercontent.com/9084751/104753085-cd1e0500-5757-11eb-85ba-de37f84c5f16.png)

Reply:
My bad, I should make the difference clear. The following figure illustrates the layout of standard 10-20 systems which is close to its (64+3)-channel version:
![montage-standard1020-for64](https://user-images.githubusercontent.com/14837418/104809507-bc898f80-5828-11eb-9432-2ae460f837d4.png)
And the second figure represents the standard 10-20 system for (62+3) channel (I find it during an experiment on SEED dataset to compare our method with the state-of-the-art, and I notice it is widely compared):
![montange-standard1020-for62](https://user-images.githubusercontent.com/14837418/104809510-c01d1680-5828-11eb-9842-422e70294787.png)

Reply:
It looks like CB1 is identical to POO7 in the 10-05 system (https://github.com/sappelhoff/eeg_positions/blob/master/data/images/1005.png)

Reply:
> It looks like CB1 is identical to POO7 in the 10-05 system

yes, it certainly looks like it. 

@tczhangzhi you could perhaps verify by reading this article: https://www.sciencedirect.com/science/article/abs/pii/S1388245700005277

Reply:
k! thanks!

Reply:
Yes, m afraid you guys are right! Learn a lot, thanks for your time~ Then, this PR is only a subset of standard-1005, and by renaming the channel and use `on_missing` parameter we can achieve the same thing. So I will close this PR~ If you have any other questions feel free to reopen it, thanks.

Reply:
Cool that this issue could be solved and good luck